### PR TITLE
manifest: nrfxlib: 802.15.4 multiprotocol support

### PR DESCRIPTION
**Pull request by jciupis**
MPSL library is now capable of configuring the nRF IEEE 802.15.4 radio driver to support multi-protocol use cases.

Related to nrfxlib [PR#234](https://github.com/nrfconnect/sdk-nrfxlib/pull/234)

Signed-off-by: Jedrzej Ciupis <jedrzej.ciupis@nordicsemi.no>

### CODE REVIEW

**Comment by jciupis** [comment]
The CI fails due to missing symbols that are defined in the upstream repos. Until they're merged to NCS it's not going to change. Related PRs:
https://github.com/zephyrproject-rtos/hal_nordic/pull/49
https://github.com/zephyrproject-rtos/zephyr/pull/27460

File: west.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: aeeb6a08afd0529c366728183367adf009104a4a
+      revision: pull/234/head
```
Comment (carlescufi) [inline west.yml:100]: b69c5a7c25007fbc111a0cbda6e2a6a945a6ae9a

Comment (jciupis) [reply west.yml:100]: You probably meant to replace the revision [here](https://github.com/zephyrproject-rtos/zephyr/pull/27460/files#diff-4ae7a6d2c692e706a33f7a71d3eb25d2R59)

Comment (carlescufi) [reply west.yml:100]: I did, sorry.

**Comment by joerchan** [comment]
Since the nrfxlib PR is a duplicate, can this be closed then?

**Comment by jciupis** [comment]
Yes, the changes introduced in the nrfxlib PR have already been merged, yielding this PR obsolete

**Pull request closed without merging**